doc/nrf/libraries/networking/nrf_cloud_pgps.rst

```diff
@@ -86,6 +86,21 @@ The P-GPS library requires a storage location in the flash memory for storing th
 There are three ways to define this storage location:
 
 * To use a dedicated partition, enable the :kconfig:option:`CONFIG_NRF_CLOUD_PGPS_STORAGE_PARTITION` option.
+
+  By default, this partition is stored in the main SoC flash.
+  This partition can optionally be located in external flash for the nRF9160 development kit version 1.0.1 and higher.
+  This conserves space in the main flash for storing code or other data.
+  Currently, you cannot combine storing P-GPS data in external flash with full modem FOTA.
+
+  To enable this, add the following parameter to your build command:
+  
+  ``-DOVERLAY_CONFIG=overlay_pgps_ext_flash.conf``
+  
+  Also, specify your development kit version by appending it to the board name.
```

Comment:
What happens if you don't specify this? Including that detail could include the chances of someone spotting this line after encountering that issue

Had literally that exact thing happen to me with the BLE sensor gateway. Turns out I needed to specify the DK version, and I had missed the line stating "this will fail if you don't specify it", but them explicitly stating "this will fail" is what pointed that line out to me

Reply:
Good idea.